components/Overview/index.tsx

```diff
@@ -136,19 +136,42 @@ class Overview extends React.Component<Props> {
                   ))}
                 </p>
 
-                {selectedPool.website && (
+                {(selectedPool.website ||
+                  selectedPool.email ||
+                  (selectedPool.additionalContactInfo && selectedPool.additionalContactInfo.length > 0)) && (
                   <p>
-                    <strong>Contact the Asset Originator</strong>
-                    <br />
-                    Interested in learning more?
-                    <br />
-                    <a href={selectedPool.website} target="_blank">
-                      {new URL(selectedPool.website).hostname}
-                    </a>
-                    <br />
-                    <a href={`mailto:${selectedPool.email}`} target="_blank">
-                      {selectedPool.email}
-                    </a>
+                    <strong>Interested in investing or want to learn more?</strong>
+                    {selectedPool.website && (
+                      <>
+                        <br />
+                        <a href={selectedPool.website} target="_blank">
+                          {new URL(selectedPool.website).hostname}
+                        </a>
+                      </>
+                    )}
+                    {selectedPool.website && (
+                      <>
+                        <br />
+                        <a href={`mailto:${selectedPool.email}`} target="_blank">
+                          {selectedPool.email}
+                        </a>
+                      </>
+                    )}
+                    {selectedPool.additionalContactInfo?.map((ci, i) =>
+                      ci.link ? (
+                        <React.Fragment key={ci.label + i}>
```

Comment:
I personally think the short syntax, `<>{...children}</>`, is cleaner. What do you think? 

https://reactjs.org/docs/fragments.html#short-syntax